k4FWCore/scripts/k4run

```diff
@@ -45,62 +45,65 @@ FILTER_GAUDI_PROPS = [
     "AutoRetrieveTools",
 ]
 
-# ---------------------------------------------------------------------
 
-seen_files = set()
-option_db = {}
+def add_arguments(parser, app_mgr):
+    """
+        Add arguments to the parser for all properties of all configurables in the application manager
 
+        :param parser: the parser to add the arguments to
+        :param app_mgr: the application manager to get the properties from
+
+        :return: a dictionary mapping the argument name to the configurable it belongs to
```

Comment:
Point for discussion here: Which docstring style do we want to use in Key4hep? I don't think we have consistently decided this (or even discussed this). In podio I opted for  [google's style for docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

Reply:
I don't mind. We can discuss this where some default configuration would be placed (https://github.com/key4hep/key4hep-dev-utils).